packages/miniflare/src/plugins/assets/index.ts

```diff
@@ -87,7 +87,12 @@ export const ASSETS_PLUGIN: Plugin<typeof AssetsOptionsSchema> = {
 				],
 			},
 		};
-
+		// Set defaults:
+		const assetConfig = {
```

Comment:
do we have a type for assetConfig? Might be a good idea to have one

Reply:
whoops that was duplicated anyway because asset-worker does applyConfigurationDefaults as well.